pkg/config/draftconfig.go

```diff
@@ -188,6 +202,28 @@ func (d *DraftConfig) ApplyDefaultVariables() error {
 				variable.Value = defaultVal
 			}
 
+			if len(variable.ActiveWhenConstraints) > 0 {
```

Comment:
this logic is duplicated on 283. seems like it should be a method on ActiveWhenConstraint

Reply:
I merged the logic into the `CheckActiveWhenConstraint` function of the Draftconfig